packages/components/src/templates/next/components/complex/Infobar/Infobar.tsx

```diff
@@ -55,73 +97,67 @@ const Infobar = ({
   const hasPrimaryCTA = !!buttonLabel && !!buttonUrl
   const hasSecondaryCTA = !!secondaryButtonLabel && !!secondaryButtonUrl
 
-  return (
-    <section
-      className={compoundStyles.outerContainer({ layout: simplifiedLayout })}
-    >
-      <div
-        className={compoundStyles.innerContainer({
-          layout: simplifiedLayout,
-        })}
-      >
-        <div
-          className={compoundStyles.headingContainer({
-            layout: simplifiedLayout,
-          })}
-        >
-          <h2 className={compoundStyles.title({ layout: simplifiedLayout })}>
-            {title}
-          </h2>
+  const styles = createInfobarStyles({
+    layout: simplifiedLayout,
+    colorScheme: variant,
+  })
 
-          {description && (
-            <p
-              className={compoundStyles.description({
-                layout: simplifiedLayout,
-              })}
-            >
-              {description}
-            </p>
-          )}
-        </div>
-
-        {(hasPrimaryCTA || hasSecondaryCTA) && (
-          <div
-            className={compoundStyles.buttonContainer({
-              layout: simplifiedLayout,
-            })}
-          >
-            {hasPrimaryCTA && (
-              <LinkButton
-                href={getReferenceLinkHref(
-                  buttonUrl,
-                  site.siteMap,
-                  site.assetsBaseUrl,
-                )}
-                size={simplifiedLayout === "homepage" ? "lg" : "base"}
-                LinkComponent={LinkComponent}
-                isWithFocusVisibleHighlight
-              >
-                {buttonLabel}
-              </LinkButton>
-            )}
+  // NOTE: Should not have dark mode on non-homepage for now
+  const buttonColorScheme =
+    simplifiedLayout === "default"
+      ? "default"
+      : variant === "dark"
+        ? "inverse"
+        : "default"
```

Comment:
[nit] could we simplify this to `simplifiedLayout === "homepage" && variant === "dark" ? "inverse" : "default"`